lib/instrumentation/mongodb/common.js

```diff
@@ -264,3 +264,16 @@ function getInstanceAttributeParameters(shim, obj) {
     }
   }
 }
+
+/**
+ * Parses mongo address that accounts for IPv6
+ *
+ * @param {string} address mongo address string
+ * @returns {Array} host/port of address string
+ */
+common.parseAddress = function parseAddress(address) {
+  const lastColon = address.lastIndexOf(':')
+  const host = address.slice(0, lastColon)
+  const port = address.slice(lastColon + 1)
+  return [host, port]
+}
```

Comment:
> I think I'd like to see a test just to make sure that, for example, in ipv6 ::1 won't get parsed as 1 being the port number

This would totally happen if `::1` was being passed in but in our limited tests it always has port.  But this code is getting implicitly tested as it's needed to get passing tests on mongodb in node 18

Reply:
I don't believe we have tests where we have connexion strings like `mongodb://127.0.0.1`, which seem to be allowed. This would have broken anyway with IPv4, so I'm surprised our customers haven't complained.

Reply:
Also, if you look in this file https://github.com/newrelic/node-newrelic/blob/main/test/versioned/mongodb/common.js#L64 we have mongo connection strings and when running in node 18 those are listening on ipv6 not ipv4 even though we provide `mongodb://localhost:<port>`

Reply:
Okay, fine, guess it's not a problem.